java/jarviz/Portfile

```diff
@@ -57,11 +57,21 @@ destroot {
 
     # Add symlink to the script
     ln -s ../share/java/${name}/bin/jarviz ${destroot}${prefix}/bin
+
+    # Install shell completions
+    set comp_path ${destroot}${prefix}/share/bash-completion/completions/${name}
+    xinstall -d [file dirname ${comp_path}]
+    system -W ${worksrcpath}/bin "./${name} generate-completion > ${comp_path}"
 }
 
 notes "
-    Add the following command to your shell configuration file for command completion support:
+    Bash completion is automatically loaded for bash if bash-completion port is installed.
+    To use bash completion in zsh, add the following command to your .zshrc:
 
-        source <(jarviz generate-completion)
+        autoload -U +X bashcompinit && bashcompinit
```

Comment:
I just realized that it shouldn't be necessary for users to add this line, because the completion script itself already contains this line, right?

```suggestion
```

Reply:
Notes updated